src/Sylius/Bundle/CoreBundle/Resources/config/services/listeners.xml

```diff
@@ -86,5 +86,10 @@
             <argument>_sylius.cart</argument>
             <tag name="kernel.event_subscriber" />
         </service>
+        <service id="sylius.listener.order_customer_ip" class="Sylius\Bundle\CoreBundle\EventListener\OrderCustomerIpListener">
+            <argument type="service" id="sylius.customer_ip_assigner" />
+            <argument type="service" id="request_stack" />
+            <tag name="kernel.event_listener" event="sylius.order.pre_summary" method="assignCustomerIpToOrder" />
```

Comment:
The event dispatched on order complete should be renamed from `summary` to `complete` (separate PR).